internal/app/init/pkg/system/runner/process/log/log.go

```diff
@@ -8,14 +8,14 @@ import (
 	"context"
 	"fmt"
 	"os"
-	"path"
+	"path/filepath"
```

Comment:
should be no-op for Unix filenames, but seems a bit cleaner as we're combining filepaths